### PR TITLE
Start apply faq updates

### PR DESCRIPTION
**Pull request by drew-usds**

### CODE REVIEW

File: pages/apply.html [hunk 1]
```diff
@@ -20,6 +20,24 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
       <div class="site-c-section-heading__text site-c-section-heading__text--big usa-prose">
         {{ content.page_title.heading_text | replace: 'site.baseurl', site.baseurl | markdownify}}
       </div>
+      <div>
+        <h2>Tour of service model</h2>
+          <p>
+            foo
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
USDS operates on a tour-of-service model, with most people serving for one or two years, and often options to extend up to four years. We'll consider some shorter tours, and time commitments are not binding.
```

File: pages/apply.html [hunk 2]
```diff
@@ -20,6 +20,24 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
...
+          </p>
+        <h2>Requirements</h2>
+          <p>
+            foo
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
You must be a citizen of the United States to work at USDS. On top of that, you'll need to pass a background check and a drug test. During the COVID-19 pandemic, most people at USDS are working under a distributed model, from many locations across the country. During normal times, most of the teams we work with are in the D.C. metro area, so we ask that you make D.C. or the surrounding area your base during the work week.
```

File: pages/apply.html [hunk 3]
```diff
@@ -20,6 +20,24 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
...
+          </P>
+          <h2>What you'll get</h2>
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
          <h2>Compensation and Benefits</h2>
```

File: pages/apply.html [hunk 4]
```diff
@@ -20,6 +20,24 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
...
+            <p>
+              foo
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
Salaries for digital service experts at USDS vary, but don't exceed $170,800, determined by your experience and qualifications. We provide health care and other benefits, such as retirement savings account eligibility. For a wealth of information and different benefit plans offered look on the [OPM website](https://www.opm.gov/healthcare-insurance/healthcare/plan-information/plans/).
```

File: pages/apply.html [hunk 5]
```diff
@@ -20,6 +20,24 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
...
+            </p>
+          <h2>Hiring process</h2>
+            <p>
+              foo
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
Our talent team will be with you every step of the way through your government hiring odyssey. First, complete the application and attach your resume (and portfolio, if you're a designer!). A member of the talent team will be in touch within two weeks after you apply. People who are invited to the interview process will connect with a member of the talent team over the phone, after which you'll complete interviews to assess for skills important to our communities of practice. If selected, your talent partner will work with you to match you to a project.

Onboarding to USDS can take several weeks, and sometimes longer depending on the background check. Your talent partner will keep you up-to-date throughout.

If you aren't selected to join USDS, you can apply again after one year with an updated resume.
```

File: pages/apply.html [hunk 6]
```diff
@@ -20,6 +20,26 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
...
+        </p>
+        <p>
+          <strong>Compensation and benefits:</strong>
+Salaries for digital service experts at USDS vary, but don't exceed $170,800, determined by your experience and qualifications. We provide health care and other benefits, such as retirement savings account eligibility. For a wealth of information and different benefit plans offered look on the <a href="https://www.opm.gov/healthcare-insurance/healthcare/plan-information/plans/">OPM website</a>.
+        </p>
+        <p>
+          <strong>Hiring process:</strong>
+          Our talent team will be with you every step of the way through your government hiring odyssey. First, complete the application and attach your resume (and portfolio, if you're a designer!). A member of the talent team will be in touch within two weeks after you apply. People who are invited to the interview process will connect with a member of the talent team over the phone, after which you'll complete interviews to assess for skills important to our communities of practice. If selected, your talent partner will work with you to match you to a project.</p>
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
          Our talent team will be with you every step of the way through your government hiring odyssey.
          <ol>
          <li>Complete the application and attach your resume (and portfolio, if you're a designer!).</li>
          <li>A member of the talent team will be in touch within two weeks after you apply.</li>
          <li>People invited to the interview process will connect with a member of the talent team over the phone.</li>
          <li>Complete interviews to assess for skills important to our communities of practice.</li>
          <li>If selected, your talent partner will work with you to match you to a project.</li>
          </ol>
          </p>
```

File: pages/apply.html [hunk 7]
```diff
@@ -20,6 +20,34 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
       <div class="site-c-section-heading__text site-c-section-heading__text--big usa-prose">
         {{ content.page_title.heading_text | replace: 'site.baseurl', site.baseurl | markdownify}}
       </div>
+      <div class="usa-prose">
+        <p>USDS operates on a tour-of-service model, with most people serving for one or two years, and often options to extend up to four years. We'll consider some shorter tours, and time commitments are not binding.
+        </p>
+        <p>
+          <strong>Requirements:</strong>
+          You must be a citizen of the United States to work at USDS. On top of that, you'll need to pass a background check and a drug test. During the COVID-19 pandemic, most people at USDS are working under a distributed model, from many locations across the country. During normal times, most of the teams we work with are in the D.C. metro area, so we ask that you make D.C. or the surrounding area your base during the work week.
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
          To work at USDS, you must be a U.S.citizen, and pass a background check and a drug test. During the COVID-19 pandemic, most people at USDS are working remotely from many locations across the country. During normal times, we ask that you make D.C. or the surrounding area your base during the work week.
```

File: pages/apply.html [hunk 8]
```diff
@@ -20,6 +20,34 @@ <h1 class="site-c-section-heading__heading site-c-section-heading__heading--h1">
...
+        </p>
+        <p>
+          <strong>Compensation and benefits:</strong>
+Salaries for digital service experts at USDS vary, but don't exceed $170,800, determined by your experience and qualifications. We provide health care and other benefits, such as retirement savings account eligibility. For a wealth of information and different benefit plans offered look on the <a href="https://www.opm.gov/healthcare-insurance/healthcare/plan-information/plans/">OPM website</a>.
```
Comment (bernars-usa) [inline pages/apply.html]:
```suggestion
Salaries at USDS vary, but don't exceed $170,800, determined by your experience and skills. We provide health care and other benefits, such as retirement savings accounts. For more information, see the <a href="https://www.opm.gov/healthcare-insurance/healthcare/plan-information/plans/">OPM website</a>.
```

**Comment by bernars-usa** [comment]
## FAQs we can 🔪 

For sure:

- Getting Hired
  - What benefits will I receive?
  - How much of a time commitment will I make if I accept one of these positions?
  - If I apply, how long will it take to go through the process?
  - When will I hear from USDS in this process?
  - Is USDS an equal opportunity employer?
  - So I didn't get the job? What's next?
- Relocation
  - Do I have to live in Washington, D.C.?

Debatable:

- Getting Hired:
  - What is required to work for the U.S. Digital Service?
  - What kind of salary might I make?

@mriverasl3 - Could you take a look at these and let me know if the new text we're adding to the Apply page is sufficient for these "questions"?

**Comment by drew-usds** [comment]
@bernars-usa finally got a chance to update the  layout to match some existing pages. Since I sat on this for a minute, I also made some text edits after coming back to this with "I am new to working in this space" eyes. Let me know if I took it _too_ far. I moved the content out of the template and into `/content/apply.yml`, so scope the below screenshot if that's easier to read than the .yml. I can also spin up a staging version if that's more helpful.

Screen Shot 2020-12-22 at 1 53 55 AM

**Comment by mriverasl3** [comment]
i like the content and layout

File: pages/apply.html (hunk 7 above)
Comment (drew-usds) [reply pages/apply.html]: Whoops missed this in the last commits, adding into another batch!

File: pages/apply.html (hunk 8 above)
Comment (drew-usds) [reply pages/apply.html]: Whoops accidentally left this out of the previous commits, adding in the next one!

File: _content/apply.yml [hunk 9]
```diff
@@ -1,10 +1,29 @@
...
-      The best of technology.
-      The best of government.
-      And we want you.
+  heading_text: "USDS operates on a tour-of-service model with a maximum term of four years. While most people serve for one or two years, we'll consider some shorter tours as well. Time commitments are not binding."
   heading_button_text: "Read our hiring FAQ"
+page_body:
+  hiring_process: |-
+    **Hiring process**
+
+    1. Complete the below application and attach your resume (and portfolio, if you're a designer!).
```
Comment (bernars-usa) [inline _content/apply.yml]:
```suggestion
    1. Complete the application below and attach your resume (and portfolio, if you're a designer!).
```

File: _content/apply.yml [hunk 10]
```diff
@@ -1,10 +1,29 @@
...
+
+    Onboarding to USDS can take several weeks, and sometimes longer depending on completion of a required background check. Our talent team will be with you every step of the way, and your talent partner will keep you up-to-date throughout the process.
+
+    If you aren't selected to join USDS, you can apply again after one year with an updated resume.
+  requirements: |-
+    **Requirements**
+
+    To work at USDS, you must be a U.S. citizen, and pass a background check and a drug test.
+
+    During the COVID-19 pandemic, most people at USDS are working remotely from many locations across the country. During normal times, we ask that you make D.C. or the surrounding area your base during the work week.
```
Comment (bernars-usa) [inline _content/apply.yml]:
```suggestion
    During the COVID-19 pandemic, most people at USDS are working remotely from many locations across the country. During normal times, we ask that you make Washington, D.C. or the surrounding area your base during the work week.
```

**Review by bernars-usa — DISMISSED** [review]
Just a couple of very minor tweaks, then I think we can ship this sucker! Thanks a million, Drew! ❤️

**Pull request merged**